README.rst

```diff
@@ -1,23 +1,150 @@
-kinto2xml
----------
+#########
+amo2kinto
+#########
 
-Generate a blocklist.xml file from Kinto collections.
+
+Installation
+============
+
+To install the release, just use::
+
+    pip install amo2kinto
+
+
+To install the development environment, just run::
+
+    make install
+
+
+System dependencies
+-------------------
+
+In order to build ``amo2kinto`` dependencies you may need the following libraries:
+
+- Building **lxml** will require: ``libxml2-dev libxslt-dev python-dev``
+- Building **cryptography** will require: ``build-essential libffi-dev libssl-dev python-dev``
+
+If you have got error messages telling you a header file is missing, please install them:
+
+- ``libxml/xmlversion.h: File not found``
+- ``ffi.h: File not found``
+- ``ssl.h: File not found``
+
+
+Importing blocklists from AMO into Kinto
+========================================
+
+The `addons-server <https://github.com/mozilla/addons-server/>`_ is
+able to export its blocklists database in JSON.
+
+You can use the ``json2kinto`` script to load this database into a
+kinto server::
+
+    json2kinto --server http://localhost:8888/v1 \
+               --amo-server https://addons.mozilla.org/  \
```

Comment:
Yes there are [plenty of them](https://github.com/mozilla/addons-server/#addons-server):

- http://addons-dev.allizom.org/
- https://addons.allizom.org/
- https://addons.mozilla.org/